lib/cdo/contact_rollups.rb

```diff
@@ -39,9 +39,19 @@ class ContactRollups
   # Table name of destination working table
   DEST_TABLE_NAME = "contact_rollups_daily".freeze
 
-  # List of valid PD courses. If more courses get added, need to update this list and also schema in Pardot. We
+  # Set of valid PD courses. If more courses get added, need to update this list and also schema in Pardot. We
   # need to filter here to known courses in the Pardot schema - we can't blindly pass values through
-  COURSE_LIST = "'CS Fundamentals','CS in Algebra','CS in Science','CS Principles','Exploring Computer Science','CS Discoveries'".freeze
+  COURSE_ARRAY = [
+    "CS Fundamentals",
+    "CS in Algebra",
+    "CS in Science",
+    "CS Principles",
+    "Exploring Computer Science",
+    "CS Discoveries"
+  ].freeze
```

Comment:
Can these course names be referenced from somewhere that defines these strings as constants? E.g., from dashboard/app/models/pd/workshop.rb?